cluster_tools/copy_volume/copy_volume.py

```diff
@@ -120,7 +115,14 @@ def run_impl(self):
             out_shape = out_shape[1:]
 
         compression = task_config.pop("compression", "gzip")
-        dtype = str(ds_dtype) if self.dtype is None else self.dtype
+
+        if self.dtype is None:
+            dtype = str(ds_dtype)
+        elif self.int_to_uint and np.issubdtype(ds.dtype, np.signedinteger):
```

Comment:
We don't have any guarantees here that the dtype is represented as `int8`, it might for example be written as `<i1`.
That's why the dtype mapping is done in line 126. So it's better if we remove the branch here and instead add after line 126:
```
if self.int_to_uint:
   dtype = "u" + ds.dtype
```
Also please remove the `and np.issubdtype(...)` and instead add a check that we have a signedinteger type (see also my previous comment).